release/release_gold.sh

```diff
@@ -135,7 +135,7 @@ function prepare_branch_to_commit {
 }
 
 function prepare_next_release_branch {
-	if [[ "${_PRODUCT_VERSION}" != *q* ]]
+	if [[ "${_PRODUCT_VERSION}" != *q* ]] || [ ! $(echo "${LIFERAY_RELEASE_PREPARE_NEXT_RELEASE_BRANCH}" | grep -i "yes") ]
```

Comment:
@lucasmiranda0, can you please check the content of the variable `LIFERAY_RELEASE_PREPARE_NEXT_RELEASE_BRANCH` first for we try to sort the `if` statement?
```
if [ ! $(echo "${LIFERAY_RELEASE_PREPARE_NEXT_RELEASE_BRANCH}" | grep -i "yes") ] ||
   [[ "${_PRODUCT_VERSION}" != *q* ]]
```